Magento2/Tests/Legacy/CodingStandardsIgnoreAnnotationUsageUnitTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Magento2\Tests\Legacy;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+class CodingStandardsIgnoreAnnotationUsageUnitTest extends AbstractSniffUnitTest
+{
+    /**
+     * @inheritdoc
+     */
+    public function getErrorList()
+    {
+        return [];
```

Comment:
There must be an error at line 7, that's where `@codingStandardsIgnoreFile` is set

Reply:
I know, i set it up empty because was expecting complains on command line when running `vendor/bin/phpunit`.
Actually, it must be a warning.